.eslintignore

```diff
@@ -0,0 +1,7 @@
+# TODO 
+# Apply ESlinting to scripts folder for this we need to remove scripts from exluded section in tsconfig.json.
+scripts/*
+jest-puppeteer.config.js
+jest-snapshot.config.js
+jest.config.js
+rollup.config.js
```

Comment:
Rest of the files, I think add a newline after ending but this and the `.eslintrc.json` does not. Could you please check once.

Reply:
added new line at end of both files